website/blog/2024-07-24-cityblock-merge-jobs.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "How Cityblock Health adopted dbt Cloud's Merge Jobs to deploy new code faster"
+description: "Cityblock Health swapped their GitHub Actions for native dbt Cloud functionality to streamline their project deployments and avoid redundant resouce consumption."
+slug: cityblock-merge-jobs
+authors: [joel_labes, nathaniel_burren]
+tags: [dbt Cloud]
+hide_table_of_contents: false
+date: 2024-07-24
+is_featured: true
+---
+
+We recently added support for a new type of job in dbt Cloud, called a **[Merge Job](/docs/deploy/merge-jobs)**. Just like a Continuous Integration job is triggered based on a new Pull Request being opened in your git repository, the Merge Job triggers once code is merged into your environment’s configured branch (e.g. `main` for the Production environment, or `uat` for the Staging environment).
```

Comment:
I think we need an intro here or to say who "we" is since its a little unclear if this is dbt or cityblock